src/plugin.ts

```diff
@@ -37,8 +42,8 @@ export async function plugin(inputs: PluginInputs) {
     }
 
     if (!isEnabled) {
-      logger.info("Plugin is disabled. Skipping.");
-      await addCommentToIssue(context, "The /gpt command is disabled. Enable it in the plugin settings.", true, "warning");
+      const log = logger.info("The /gpt command is disabled. Enable it in the plugin settings.");
+      await addCommentToIssue(context, log?.logMessage.diff as string);
       return;
     }
```

Comment:
`enabled` seems like a pointless config. Please remove that.

Reply:
Are you sure? In order to 'disable' they'd need to remove/comment out the plugin config completely from their `ubiquibot-config.yml`

I get what you mean tho either install it if you want it or don't but this way it can be configured on a per repo basis